sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/config/TransportConfig.java

```diff
@@ -52,7 +52,7 @@ public static Long getHeartbeatIntervalMs() {
         try {
             return interval == null ? null : Long.parseLong(interval);
         } catch (Exception ex) {
-            RecordLog.warn("[TransportConfig] Failed to parse heartbeat interval: " + interval);
+            RecordLog.warn("[TransportConfig] Failed to parse heartbeat interval: {}" , interval);
```

Comment:
please remove the unnecessary indent before `,`

Reply:
There is no indentation here